client/webserver/live_test.go

```diff
@@ -284,58 +284,60 @@ func (c *TCore) Sync(dex string, base, quote uint32) (*core.OrderBook, *core.Boo
 		default:
 		}
 	}
-	go func(){
-		out:
-			for {
-				select{
-				case <-time.NewTicker(feedPeriod).C:
-					// Send a random order to the order feed. Slighly biased away from
-					// unbook_order and towards book_order.
-					r := rand.Float32()
-					switch{
-					case r < 0.33:
-						// Epoch order
-						trySend(&core.BookUpdate{
-							Action: msgjson.EpochOrderRoute,
-							Order: randomOrder(rand.Float32() < 0.5, c.maxQty, c.midGap, 0.05 * c.midGap, true),
-						})
-					case r < 0.76:
-						// Book order
-						sell := rand.Float32() < 0.5
-						ord := randomOrder(sell, c.maxQty, c.midGap, 0.05 * c.midGap, false)
-						side := c.buys
-						if sell{
-							side = c.sells
-						}
-						side[ord.Token] = ord
-						trySend(&core.BookUpdate{
-							Action: msgjson.BookOrderRoute,
-							Order: ord,
-						})
-					default:
-						// Unbook order
-						sell := rand.Float32() < 0.5
-						side := c.buys
-						if sell{
-							side = c.sells
-						}
-						var tkn string
-						for tkn = range side {
-							break
-						}
-						if (tkn == "") { continue }
-						delete(side, tkn)
-
-						trySend(&core.BookUpdate{
-							Action: msgjson.UnbookOrderRoute,
-							Order: &core.MiniOrder{Token: tkn},
-						})
+	go func() {
+	out:
+		for {
+			select {
+			case <-time.NewTicker(feedPeriod).C:
+				// Send a random order to the order feed. Slighly biased away from
+				// unbook_order and towards book_order.
+				r := rand.Float32()
+				switch {
+				case r < 0.33:
+					// Epoch order
+					trySend(&core.BookUpdate{
+						Action: msgjson.EpochOrderRoute,
+						Order:  randomOrder(rand.Float32() < 0.5, c.maxQty, c.midGap, 0.05*c.midGap, true),
+					})
+				case r < 0.76:
+					// Book order
+					sell := rand.Float32() < 0.5
+					ord := randomOrder(sell, c.maxQty, c.midGap, 0.05*c.midGap, false)
+					side := c.buys
+					if sell {
+						side = c.sells
+					}
+					side[ord.Token] = ord
+					trySend(&core.BookUpdate{
+						Action: msgjson.BookOrderRoute,
+						Order:  ord,
+					})
+				default:
+					// Unbook order
+					sell := rand.Float32() < 0.5
+					side := c.buys
+					if sell {
+						side = c.sells
 					}
-				case <-ctx.Done():
-					break out
+					var tkn string
+					for tkn = range side {
+						break
+					}
+					if tkn == "" {
+						continue
+					}
+					delete(side, tkn)
+
+					trySend(&core.BookUpdate{
+						Action: msgjson.UnbookOrderRoute,
+						Order:  &core.MiniOrder{Token: tkn},
+					})
 				}
+			case <-ctx.Done():
+				break out
 			}
-		}()
+		}
+	}()
```

Comment:
These indents were changed automatically by my editor. I did not change any code in the file besides one *core.Registration -> *core.RegisterForm